From 9e291735c8e89879451c2feebda8f851dfa6e32c Mon Sep 17 00:00:00 2001
From: Daniel Albers <daniel@lbe.rs>
Date: Tue, 9 Dec 2014 17:47:50 +0100
Subject: [PATCH] Fix column spacing with xterm reset escape sequence

Change _re to also match \E(B\E[m as defined by terminfo for xterm:

$ TERM=xterm tput sgr0 | od -xa
0000000    281b    1b42    6d5b
        esc   (   B esc   [   m
---
 prettytable.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/prettytable.py b/prettytable.py
index a619659..3f6fb99 100644
--- a/prettytable.py
+++ b/prettytable.py
@@ -79,7 +79,7 @@ MARKDOWN = 13
 ORGMODE = 14
 RANDOM = 20
 
-_re = re.compile(r"\033\[[0-9;]*m")
+_re = re.compile(r"\033(\[[0-9;]*m|\(B)")
 
 
 def _get_size(text):
-- 
2.2.0.rc0.207.ga3a616c

